### ISSUE TITLE
remove homebrew from macOS install instructions

### ISSUE THREAD

**Issue by carandraug** [MicronOxford/cockpit#644]
@iandobbie The macOS instructions include installing homebrew to then install git, ftgl, and a newer python. If one is installing cockpit from PyPI, once it is there, they don't need git, macOS already comes with python installed, and cockpit no longer needs ftgl. Can we remove the whole homebrew stuff? Anyone that already uses homebrew won't need those instructions but if they don't use homebrew it seems a bit  too much to recommend installation of a package manager (no matter how useful we think it is for other things).

**Comment by carandraug** [comment]
Reading online seems like python3 on macOS is installed as part of XCode so it's not that default. But there's also the installers in https://www.python.org/downloads/mac-osx/

**Comment by iandobbie** [comment]
I believe that the default python on macos is still 2.7. With no need for ftgl we could go with the pythion.org installer but do we need to separately install freetype or does that just need the python stub code installed by pip?

**Comment by iandobbie** [comment]
I can test this with the vm that I have. I'll try it later today and update the installer code.

**Comment by carandraug** [comment]
> do we need to separately install freetype or does that just need the python stub code installed by pip?

It comes bundled on `freetype-py` which is listed as a dependency so it will be installed by pip.

**Comment by iandobbie** [comment]
I have changed the macOS install instructions to use the python.org installer. One slight wrinkle is that the hidapi required by microscope need the xcode command line tools so I had to include a bit about that. If the xcode stuff isn't installed the pip2 install fails but displays a dialog prompting the user to install the relevant apple tools. After the install re-running the pip3 install succeeds and generates a working setup.
